### PR TITLE
Updates Ship Ambience

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Updates ship ambience, from TG.

Adds a bunch of new ambient sounds to the ship---far more areas, like medical, the bridge, security, the detectives office, and plenty of other places all have new, unique  sounds that they can play when you enter the area.

This also updates the base ship ambience (the "buzz'")---just having a loud HUMMMMMMMMM wasn't ideal and just got on one's nerves---it's been updated to be more of a quiet whooshing sound as if ventilation in an office is on; it's a lot easier on the ears and doesn't feel so dated.

Code wise, this purges the unused `music` var.

This generally makes the ship feel more "alive" and helps increased immersion or give a "feel" to an area.

Of course, as always, if you don't like it, you can just turn it off.

This does remove traitor.ogg; it's a massive 1.5 MB sound file that was previously used for entering the chapel and space--it was so massive that it'd often stutter you when you first entered....not only that, the song is over 5 minutes long; you're typically not going to get a chance to finish the song before triggering another ambient noise.

:cl: Fox McCloud
add: Ship base ambient buzz has been updated to sound more like ventilation airflow as opposed to a buzz
add: Area ambience has also been updated, there's quite a few new ambient sounds for new areas
/:cl:

### CODE REVIEW

File: code/game/area/areas.dm [hunk 1]
```diff
@@ -379,20 +373,24 @@
 
 	// Ambience goes down here -- make sure to list each area seperately for ease of adding things in later, thanks! Note: areas adjacent to each other should have the same sounds to prevent cutoff when possible.- LastyScratch
 	if(L && L.client && !L.client.ambience_playing && (L.client.prefs.sound & SOUND_BUZZ))	//split off the white noise from the rest of the ambience because of annoyance complaints - Kluys
-		L.client.ambience_playing = 1
-		L << sound('sound/ambience/shipambience.ogg', repeat = 1, wait = 0, volume = 35, channel = CHANNEL_BUZZ)
+		L.client.ambience_playing = TRUE
+		SEND_SOUND(L, sound('sound/ambience/shipambience.ogg', repeat = 1, wait = 0, volume = 35, channel = CHANNEL_BUZZ))
```
Comment (farie82) [inline code/game/area/areas.dm:377]: `TRUE` and `FALSE` here for the params.

File: code/game/area/areas.dm [hunk 2]
```diff
@@ -379,20 +373,24 @@
...
 	if(prob(35) && L && L.client && (L.client.prefs.sound & SOUND_AMBIENCE))
 		var/sound = pick(ambientsounds)
 
 		if(!L.client.played)
-			L << sound(sound, repeat = 0, wait = 0, volume = 25, channel = CHANNEL_AMBIENCE)
-			L.client.played = 1
-			spawn(600)			//ewww - this is very very bad
-				if(L && L.client)
-					L.client.played = 0
+			SEND_SOUND(L, sound(sound, repeat = 0, wait = 0, volume = 25, channel = CHANNEL_AMBIENCE))
```
Comment (farie82) [inline code/game/area/areas.dm:385]: Same here

**Comment by AffectedArc07** [comment]
> it's been updated to be more of a quiet whooshing sound as if ventilation in an office is on; it's a lot easier on the ears and doesn't feel so dated.

Thank fucking god, this noise is the first thing I disable, due to the fact it literally feels like my sound setup has broken.

**Comment by Spacemanspark** [comment]
Can we have an option to switch between the hummm sound and the new sound you're putting in?
I personally find it jarring when the sound isn't there, and it's never actually bothered me--but I certainly do understand why it might be bothersome to a few.
Otherwise, this is a nice PR.

**Comment by Fox-McCloud** [comment]
>Can we have an option to switch between the hummm sound and the new sound you're putting in?

Nope. Not going to happen. Sorry.

**Comment by Crazylemon64** [comment]
Approved by Neca

**Pull request merged**